src/components/PasswordEmail/PasswordEmail.tsx

```diff
@@ -0,0 +1,79 @@
+import React, {useState, useEffect} from 'react'
+import {Field, Form} from 'react-final-form'
+import './PasswordEmail.scss'
+
+interface IProps{
+    passwordComponent: boolean;
+    submitBtn: any;
+    input: string;
+    validate: any;
+}
+
+const PasswordEmail:React.FC<IProps> = ({submitBtn, passwordComponent, input, validate}) => {
```

Comment:
Consider some more descriptive name, since this is just password field and "PasswordEmail" sounds a bit confusing